src/Get.tsx

```diff
@@ -89,7 +90,7 @@ export interface GetProps<TData, TError> {
    * to fetch from an entirely different URL.
    *
    */
-  base?: string;
+  base: string;
```

Comment:
This props needs to be optional if we want to use `<Get />` with specify any `base`, since you have a defaultProps to always have a value, you can simply use the typescript bang operator:


```typescript
this.props.base! // adding ! tell to typescript that this this.props.base can't be undefined
```

A very nice article about this -> https://levelup.gitconnected.com/ultimate-react-component-patterns-with-typescript-2-8-82990c516935

(and normally the last version of typescript can deal with this defaultProps case, but I don't try yet)